core/chains/evm/txmgr/models.go

```diff
@@ -42,6 +41,8 @@ type (
 	EvmTxmClient              = txmgrtypes.TxmClient[*big.Int, common.Address, common.Hash, common.Hash, *evmtypes.Receipt, evmtypes.Nonce, gas.EvmFee, EvmAccessList]
 )
 
+var NewEvmTxm = NewTxm[*big.Int, *evmtypes.Head, common.Address, common.Hash, common.Hash, *evmtypes.Receipt, evmtypes.Nonce, gas.EvmFee, EvmAccessList, *assets.Wei]
```

Comment:
curious why did you choose to go `var NewEvmTxm` instead of `type NewEvmTxm` ?

Reply:
`NewTxm` is a function not a type so aliasing the function would be done using `var`

Reply:
It is a function. Looking at it, does it make sense to rename NewEvmTxm to NewEvmTxmFn, for clarity?

Reply:
You could avoid the risks of this mutable var by using a wrapper function to forward the call instead :shrug:

Reply:
It _might_ be more verbose, but it could end up more readable if the generic types can be inferred. Either way, you get named vars instead of just blank types.